tutorials/sample-datasets/plot_sleep.py

```diff
@@ -118,6 +123,13 @@
 events_train, _ = mne.events_from_annotations(
     raw_train, event_id=annotation_desc_2_event_id, chunk_duration=30.)
 
+# keep last 30-min wake events before sleep and first 30-min wake events after
+# sleep
+sleep_stage_inds = np.where(events_train[:, 2] > 1)[0]
+offset = int((30 * 60) / 30)
```

Comment:
The units here / reasons for the given numbers are difficult to parse. The 60 makes sense because it's sec/min, the first 30 makes sense because it's min, but then the last 30 is ... ? Probably making use of some informative variable names above (possibly having to do with the time between samples?) and reusing them here could fix this problem.

Also `int(x_int / y_int)` is less clean than `x_int // y_int`